### PR TITLE
[11.0][FIX] stock_request_tier_validation

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Fix to be included after [server-ux/76](https://github.com/OCA/server-ux/pull/76) is merged.

cc ~ @Eficent

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]
Tested on runbot.

image

**Review by rousseldenis — APPROVED** [review]
Tested too !

File: oca_dependencies.txt [hunk 1]
```diff
@@ -3,5 +3,8 @@
 product-attribute
 purchase-workflow
 web
-server-ux
+# server-ux
 account-analytic
+
+# To be removed before merge, testing purposes
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: could you update this? Thanks!

**Review by LoisRForgeFlow — APPROVED** [review]

**Comment by rousseldenis** [comment]
@lreficent to avoid bot putting ready label, make a review with request change. The needs fixing label does not have influence

**Pull request closed without merging**